cli/cmd/version.go

```diff
@@ -13,26 +13,44 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	cliFlag     = "cli"
+	clusterFlag = "cluster"
+)
+
 var (
-	OdigosVersion string
-	OdigosCommit  string
-	OdigosDate    string
+	OdigosVersion        string
+	OdigosClusterVersion string
+	OdigosCommit         string
+	OdigosDate           string
 )
 
 // versionCmd represents the version command
 var versionCmd = &cobra.Command{
 	Use:   "version",
 	Short: "Print odigos version.",
 	Run: func(cmd *cobra.Command, args []string) {
-		short_flag, _ := cmd.Flags().GetBool("short")
+		short_flag, _ := cmd.Flags().GetBool(cliFlag)
```

Comment:
The conventions for this project are camelCase 
```suggestion
		shortFlag, _ := cmd.Flags().GetBool(cliFlag)
```